packages/app-degree-pages/src/core/utils/string-utils.js

```diff
@@ -1,6 +1,7 @@
 // @ts-check
 
 function toTitleCase(str) {
+  if (typeof str !== "string") return console.error("Invalid string", str);
```

Comment:
return statement like this I try to avoid since they are easier to overlook. I thought we had a linting rule that prefers we do not do this shorthand. 

That being said since it is at the top of the function it doesn't bother me too much.